references/classification/train_pytorch.py

```diff
@@ -28,6 +28,27 @@
 from utils import plot_recorder, plot_samples
 
 
+def export_to_onnx(model, name, val_loader_batch):
+    """Export the model to ONNX format."""
+    dummy_input = val_loader_batch[0]
+    torch.onnx.export(
+        model,
+        dummy_input,
+        f"{name}.onnx",
+        input_names=['input'],
+        output_names=['logits'],
+        dynamic_axes={'input': {0: 'batch_size'}, 'logits': {0: 'batch_size'}},
+        export_params=True, opset_version=13, verbose=False
+    )
+    print(f"Model exported to {name}.onnx")
+    try:
+        from onnxruntime.quantization import quantize_qat
+        quantize_qat(f"{name}.onnx", f'{name}.quant.onnx')
+        print(f"Quantized model saved to {name}.quant.onnx")
+    except ImportError:
+        print("ONNX Runtime is not installed. Skipping quantization.")
```

Comment:
Wince you put onnx in the requirements, I think we don't need the try except and we can simply import it at the beginning of the file